conda/cli/main_create.py

```diff
@@ -3,10 +3,36 @@
 
 from __future__ import absolute_import, division, print_function, unicode_literals
 
+from logging import getLogger
+from os.path import isdir
+
+from .common import confirm_yn
 from .install import install
-from ..gateways.disk.delete import delete_trash
+from ..base.context import context
+from ..common.path import paths_equal
+from ..exceptions import CondaValueError
+from ..gateways.disk.delete import delete_trash, rm_rf
+from ..gateways.disk.test import is_conda_environment
+
+log = getLogger(__name__)
 
 
 def execute(args, parser):
+    if is_conda_environment(context.target_prefix):
+        if paths_equal(context.target_prefix, context.root_prefix):
+            raise CondaValueError("The target prefix is the base prefix. Aborting.")
+        confirm_yn("WARNING: A conda environment already exists at '%s'\n"
```

Comment:
TODO: Incorporate `conda.common.io.timeout` into the `confirm_yn()` function before this PR is merged.

Reply:
Done.